server/plugin/plg_backend_s3/index.go

```diff
@@ -12,12 +19,6 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
 	. "github.com/mickael-kerjean/filestash/server/common"
-	"io"
```

Comment:
Please don't reformat the existing code

Reply:
We'd better to follow best practices.
[import-group-ordering](https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering)
[Why group and order imports in Go?](https://bongnv.com/blog/2022-03-09-why-group-imports-in-go/)

Reply:
Yes but as of this commit, please don't reformat the existing code, we're either talking about a PR that fix an issue or a PR that reformat some code, not both